aion/web/channels/subject_channel.ex

```diff
@@ -21,16 +21,18 @@ defmodule Aion.SubjectChannel do
   end
 
   def handle_in("new:answer", %{"room_id" => room_id, "answer" => answer}, socket) do
-    username = socket.assigns.current_user.name
-    evaluation = ChannelMonitor.new_answer(room_id, answer, username)
-    send_feedback socket, evaluation
+     username = socket.assigns.current_user.name
+     evaluation = ChannelMonitor.new_answer(room_id, answer, username)
+     send_feedback socket, evaluation
 
-    if evaluation == 1.0 do
-      send_user_list(socket, room_id)
-      send_new_question(socket, room_id)
-    end
-
-    {:noreply, socket}
+     case evaluation do
+       1.0 ->
+         send_user_list(socket, room_id)
+         send_new_question(socket, room_id)
+         {:noreply, socket}
+       _ ->
+         {:reply, :error, socket}
```

Comment:
Error suggests that there is a problem with the program's flow, which is not true, because even if the user answers incorrectly, it's still proper from the program's point of view. I would save atom's like :error for situations when something unexpected appears. (something like :wronganswer would do much better imo)